src/v/coproc/tests/utils/event_publisher.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+#pragma once
+#include "config/configuration.h"
+#include "coproc/tests/utils/coprocessor.h"
+#include "coproc/tests/utils/wasm_event_generator.h"
+#include "coproc/types.h"
+#include "kafka/client/client.h"
+#include "kafka/protocol/produce.h"
+#include "model/record_batch_reader.h"
+
+#include <seastar/core/sstring.hh>
+
+#include <vector>
+
+namespace coproc::wasm {
+using publish_result
```

Comment:
Maybe struct is a better option here since tuple even have a specific name already?